lite/base_verifier.go

```diff
@@ -14,7 +14,11 @@ var _ Verifier = (*BaseVerifier)(nil)
 // later, requiring sufficient votes (> 2/3) from the given valset.
 // To verify blocks produced by a blockchain with mutable validator sets,
 // use the DynamicVerifier.
-// TODO: Handle unbonding time.
+//
+// NOTE: Verifier as a supported interface is deprecated, it may be reasonable
```

Comment:
Note: Please mark the interface (in types.go) as deprecated (see: https://github.com/golang/go/wiki/Deprecated)

Reply:
I think we should just delete it straight away